packages/docs-data/compile-docs-data.mjs

```diff
@@ -92,18 +105,5 @@ function transformDocumentalistData(key, value) {
         // reverse the list so highest version is first (easier indexing)
         return Array.from(majors.values()).reverse();
     }
-    if (value != null) {
-        return interpolateClassNamespace(value);
-    }
     return undefined;
 }
-
-/**
- * If value is a string, replaces `#{$ns}` in that string with the actual Blueprint class namespace.
- * Otherwise, simply returns the value.
- *
- * @param {any} value
- */
-function interpolateClassNamespace(value) {
```

Comment:
this transformation got moved to the "marked" renderer hooks lifecycle, which I think is a better place for it.

Reply:
actually, this turned out to cause [a regression in rendering the CSS API examples](https://github.com/palantir/blueprint/issues/6517) because we need to transform the _kss_ data in addition to the _markdown_ data. will fix